### ISSUE TITLE
Release to CRAN

### ISSUE THREAD

**Issue by surmann** [rdatsci/rtcl#70]
Should be release this package to CRAN to prevent the next nameclash?

**Comment by jakob-r** [comment]
Yes but this package does a lot of weird things and file system operations. I don't know how the CRAN policy is. If I you want to you can change yourself to the maintainer and go for it.
